.pre-commit-config.yaml

```diff
@@ -1095,7 +1095,7 @@ repos:
           airflow/example_dags/.*
         args:
           - "--skip"
-          - "B301,B324,B403,B404,B603"
+          - "B101,B301,B324,B403,B404,B603"
```

Comment:
@hussein-awala WDYT about add `B101` to exclusion in bandit check, since we could check it by the `ruff` ?
Tight now this rule ignored by the severity settings and cant deal with TYPE_CHECKING blocks